collectors/StatsCollector.py

```diff
@@ -53,7 +53,7 @@ def collect(self):
             for value_entry in resource.get('stat-list', {}).get('stat', []):
                 statkey = value_entry.get('statKey', {}).get('key')
                 # Normalisation of keys retrieved from API (e.g. cpu:102|usage_average -> cpu|usage_average)
-                norm_statkey = re.sub("[^a-zA-Z|_ ]+", "", statkey)
+                norm_statkey = re.sub("[^a-zA-Z|_\- ]+", "", statkey)
```

Comment:
When does the `-` come into play?

Reply:
`System Capacity|System-wide Logical Switch Ports|UsageCount`